docs/image-redactor/evaluating_dicom_redaction.md

```diff
@@ -73,7 +73,11 @@ The `DicomImagePiiVerifyEngine` class can be used to assist in ground truth labe
 
 ```python
 import pydicom
-from presidio_image_redactor import DicomImagePiiVerifyEngine
+from presidio_image_redactor import DicomImagePiiVerifyEngine, BboxProcessor
+
+# Initialize engines
+dicom_engine = DicomImagePiiVerifyEngine()
+bbox_processor = BboxProcessor()
```

Comment:
Does the user need to interact directly with this object? If it is a field in `dicom_engine` then we could call its methods through the dicom_engine, e.g. `dicom_engine.bbox_processor.get_bboxes_from_ocr_results` or expose a method e.g. `dicom_engine.get_bboxes_from_ocer_results` that would call the `bbox_processor`'s method. This could simplify the API for the user.

Reply:
Good point. There isn't really a benefit from using the `BboxProcessor` class directly from a user standpoint. In the future, we may add code within the broader image (verification) class to assist with evaluation.

I'll change this to calling `dicom_engine.bbox_processor.get_bboxes_from_ocr_results` for now and then will update with any future changes to evaluation assistance code.